azdev/params.py

```diff
@@ -115,6 +116,20 @@ def load_arguments(self, _):
 
     # endregion
 
+    with ArgumentsContext(self, 'break-change command-meta export') as c:
+        c.positional('modules', modules_type)
```

Comment:
Does this command support multi modules? What's the result of it?

Reply:
Yes, it will generate az_network_meta.json and az_monitor_meta.json if modules is ["network", "monitor"].

Reply:
OK. What if user provide multiple modules and also provide one meta_output_path argument?

Reply:
`meta_output_path` here is the folder to store the generated meta json files and if not provided, it will generate the modules' meta json under current folder.

Reply:
`azdev command-change meta-export network monitor --meta-output-path ./abc/ `
<img width="823" alt="image" src="https://user-images.githubusercontent.com/8137253/234520643-c7229f8c-dcf1-457e-81b1-6458e4dbf332.png">